src/app.rs

```diff
@@ -102,46 +102,26 @@ impl AppState {
         let db = Arc::new(Database::new("identities.db").unwrap());
         db.initialize().unwrap();
 
-        let settings = db.get_settings().expect("expected to get settings");
+        let mainnet_app_context = AppContext::new(Network::Dash, db.clone());
+        let testnet_app_context = AppContext::new(Network::Testnet, db.clone());
 
-        let mainnet_app_context = match AppContext::new(Network::Dash, db.clone()) {
-            Some(context) => context,
-            None => {
-                eprintln!(
-                    "Error: Failed to create the AppContext. Expected Dash config for mainnet."
-                );
-                std::process::exit(1);
-            }
+        let app_context = if testnet_app_context.is_some() && mainnet_app_context.is_none() {
+            testnet_app_context.clone().unwrap()
+        } else {
+            mainnet_app_context.clone().unwrap()
```

Comment:
Unnecessary. If one of the configs don't load, it can be seen in explorer.log as well as in the UI on the NetworkChooser Screen. If neither load, the app doesn't load and the terminal displays a nice error message.